src/helpers/get-thread-link.js

```diff
@@ -3,7 +3,7 @@ import slugg from 'slugg';
 import type { ThreadInfoType } from 'shared/graphql/fragments/thread/threadInfo';
 
 const getThreadLink = (thread: ThreadInfoType) => {
-  return `${thread.community.slug}/${thread.channel.slug}/${slugg(
+  return `/${thread.community.slug}/${thread.channel.slug}/${slugg(
```

Comment:
This is probably going to break in a bunch of places where we do not expect the slash to be there

Reply:
I'll remove this if we can fix the issue we're discussing on Slack. Otherwise this was just infinitely appending this string to the url :P